### PR TITLE
Rajasi | MPOC-421 Translation in code

### PR DESCRIPTION
**Pull request by rajasijaywant3129**

### CODE REVIEW

File: openelis.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+ant dist
```
Comment (nthfloor) [inline openelis.sh]: @rajasijaywant3129 what is the purpose of this script?

File: openelis/WebContent/scripts/slickgrid/slick.formatters.js [hunk 2]
```diff
@@ -49,8 +49,11 @@
     return "<span class='percent-complete-bar' style='background:" + color + ";width:" + value + "%'></span>";
   }
 
+
   function YesNoFormatter(row, cell, value, columnDef, dataContext) {
-    return value ? "Yes" : "No";
+  //var trans=MessageResources_pt.properties
```
Comment (nthfloor) [inline openelis/WebContent/scripts/slickgrid/slick.formatters.js]: can we remove these comments if they are not being used?

File: openelis/src/MessageResources_pt.properties [hunk 3]
```diff
@@ -71,7 +72,7 @@ errors.prefix=<LI>
 errors.header=<UL>
 AIDSStage.adultA=1 \= Stade A (Adulto)
 answer.yes=1 \= Sim
-answer.yesNo.description=1 \= Sim, 2 \= N\u00e3o
+answer.yesNo.description=1 \= Sim, 2 \= n�o
```
Comment (nthfloor) [inline openelis/src/MessageResources_pt.properties:75]: Does this render correctly in OpenELIS? 

I remember we were having issues with special character encodings not rendering correctly hence we used html tags

File: openelis.sh (hunk 1 above)
Comment (rajasijaywant3129) [reply openelis.sh]: By Mistakenly pushed, I will remove it

File: openelis/WebContent/scripts/slickgrid/slick.formatters.js (hunk 2 above)
Comment (rajasijaywant3129) [reply openelis/WebContent/scripts/slickgrid/slick.formatters.js]: Yes Sure

**Comment by krishnanspace** [comment]
Everything seems fine now, so merging it

**Pull request merged**